### ISSUE TITLE
ElectionResults - use same format as a way to provide candidate information before the election

### ISSUE THREAD

**Issue by evdb** [esd-org-uk/schemas#6]
Providing election results is very important, but so is providing information about elections and candidates ahead of time. This schema could be used in a slightly modified way to do that.

This could be especially useful as a standard way to publish the statements of persons nominated.

I believe that the schema could be use with the various voting related columns either empty or omitted. The `ElectionDate` would remain but be in the future. To add complete clarity a column could be added to indicate if voting results are included or not (`ElectionHasBeenHeld` perhaps).

**Comment by MikeThacker1** [comment]
I fully agree with this and understand that all the results fields/columns are optional so that data can be published before an election with the statements of persons nominated.

Perhaps the name ElectionResults could be changed to denote the schema also has a role before results come out.
